docs/tools/cloud9.md

```diff
@@ -0,0 +1,38 @@
+# Cloud9 IDE
+
+Cloud9 is an open source, web based, IDE (integrated development environment) that allows for a developer to make changes
```

Comment:
The comma before IDE is not necessary as it is not a separate item in a list. Rather open source and web-based are descriptors of the IDE.